docs/_snippets/examples/document-editor.html

```diff
@@ -115,20 +115,28 @@ <h3 style="text-align:center;">Welcome letter</h3>
 
 	/* Set the default font for the "page" of the content. */
 	.document-editor .ck-content,
-	.document-editor .ck-heading-dropdown .ck-list {
+	.document-editor .ck-heading-dropdown .ck-list .ck-button__label {
 		font: 16px/1.6 "Helvetica Neue", Helvetica, Arial, sans-serif;
 	}
 
 	/* Adjust the headings dropdown to host some larger heading styles. */
-	.document-editor .ck-heading-dropdown .ck-dropdown__panel .ck-list > .ck-list__item {
-		line-height: calc( 1.2 * var(--ck-line-height-base) * var(--ck-font-size-base) );
-		min-width: 8em;
+	.document-editor .ck-heading-dropdown .ck-list .ck-button__label {
+		line-height: calc( 1.7 * var(--ck-line-height-base) * var(--ck-font-size-base) );
+		min-width: 6em;
+	}
+
+	/* Scale down all heading previews because they're way too big to be presented in the UI.
+	Preserve the relative scale, though. */
+	.document-editor .ck-heading-dropdown .ck-list .ck-button:not(.ck-heading_paragraph) .ck-button__label {
+		transform: scale(0.8);
+		transform-origin: left;
 	}
 
 	/* Set the styles for the "Heading 1". */
```

Comment:
Oh, here is where _the "Heading N"_ comes from. I'd say, get rid of those "the"s.